analyzer/command/apk/apk_test.go

```diff
@@ -306,17 +306,39 @@ func TestResolveDependency(t *testing.T) {
 				"musl":        {},
 			},
 		},
+		"circular dependencies": {
```

Comment:
Did you confirm this test failed unless the change is applied?

Reply:
Yes!